Documentation/network/servicemesh/gateway-api/installation.rst

```diff
@@ -18,7 +18,14 @@ Prerequisites
     - `ReferenceGrant <https://gateway-api.sigs.k8s.io/api-types/referencegrant/>`_
     - `TLSRoute (experimental) <https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1alpha2.TLSRoute/>`_
 
-    .. code-block:: shell-session
+.. admonition:: Warning
```

Comment:
Similarly, let's keep the warning inside of the list?
And let's use a lowercase `w` for consistency with other admonitions.

```suggestion
    .. admonition:: warning
```